CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ### Fixed
 
+- Fixed log emission issue for ASP.NET Core 6.0 apps when bytecode
```

Comment:
Please explain what is the issue in detail stating multiple logs are emitted.

Reply:
Explained in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/3003/commits/7c21fba9d835536f6d191cc9121b48933fe8dd92